aemedge/styles/styles.css

```diff
@@ -277,7 +277,7 @@ picture {
     }
 }
 
-[data-align="left"] {
+[data-align="left"], [data-align="justify"] {
```

Comment:
adding justify because left doesn't actually work (since Google Docs uses left by default). But I needed this to be able to override globally centered text in tables.